Automattic/jetpack-boost-critical-css-gen
# Add Playwright BrowserInterface

Closes #24 

### Changes proposed in this Pull Request:
- Add a Playwright interface that extends `BrowserInterface`.
- Since Playwright does not allow passing multiple arguments to the function in `page.evaluate`, wrap the arguments into single object. Update all the functions/methods which are passed to `page.evaluate`
- Update `bin/generate.js` to use Playwright by default.

### Testing instructions:
- Run `node bin/generate.js [url]` and verify that the Critical CSS is generated. (This uses Playwright to generate Critical CSS instead of Puppeteer)
